src/status_im/keycard/recovery.cljs

```diff
@@ -126,12 +126,15 @@
 (fx/defn intro-wizard
   {:events [:multiaccounts.create.ui/intro-wizard]}
   [{:keys [db] :as cofx}]
-  (fx/merge cofx
-            {:db (-> db
-                     (update :keycard dissoc :flow)
-                     (dissoc :restored-account?))}
-            (multiaccounts.create/prepare-intro-wizard)
-            (navigation/set-stack-root :onboarding [:get-your-keys])))
+  (let [accs (get db :multiaccounts/multiaccounts)]
+    (fx/merge cofx
+              {:db (-> db
+                       (update :keycard dissoc :flow)
+                       (dissoc :restored-account?))}
+              (multiaccounts.create/prepare-intro-wizard)
+              (if (pos? (count accs))
```

Comment:
Can we use [nil punning](https://github.com/bbatsov/clojure-style-guide#nil-punning) here?

Reply:
Never mind, was on mobile, hard to type :)